christfellowshipchurch/web-app-v2
# Relevant community lineup improvements

This uses the facets from Algolia and the sub-preferences from Rock to only show the relevant lineups for a given preference. (crew, sisterhood, co-ed, etc.)

closes: #141 

TO TEST:

- Be on API Branch: `add-group-facet-filters` https://github.com/christfellowshipchurch/apollos-api/pull/244
- Go to `/community` select a preference. If any sub-preferences exist for that preference type they will show in the lineups. 
- Also, clicking the `Find your Crew` button should open the filter modal with the sub-preferences relevant to the Crew preference type.

Note: Currently, in Algolia one`Crew` and one `Sisterhood` group has sub-preferences manually inputed. See screenshots. All the other preferences should not show the lineups section since they don't currently have any groups with sub-preferences.

![image](https://user-images.githubusercontent.com/2528817/117850936-68793f80-b24b-11eb-8542-0e54b954f1cc.png)
![image](https://user-images.githubusercontent.com/2528817/117850950-6e6f2080-b24b-11eb-81dc-35027b7c4137.png)
![image](https://user-images.githubusercontent.com/2528817/117851011-7fb82d00-b24b-11eb-9f8a-ad447be6da37.png)
